Enyim.Caching/Configuration/MemcachedClientConfiguration.cs

```diff
@@ -104,181 +106,156 @@ public class MemcachedClientConfiguration : IMemcachedClientConfiguration
                 }
             }
 
-            if(!string.IsNullOrEmpty(options.KeyTransformer))
```

Comment:
The change will break the backward compatibility.

Reply:
Both Transcoder and KeyTransformer will break?

Reply:
Yes. Suggest  the following workaround.
```cs
if(!string.IsNullOrEmpty(options.KeyTransformer))
{
    //..               
}
else if (keyTransformer != null)
{
    //...
}
```